Rawl_B502/mínugud_senchusa_laigin_and_so_sís.ttl

```diff
@@ -0,0 +1,1264 @@
+    a foaf:Person;
+    irishRel:nomName "Ros";
+    irishRel:genName "Rossa";
+    owl:sameAs <http://example.com/Rawl_B502/bunad_laigen.ttl#RossaRuaid>.
+
+<#Labraid-b9133180>
+    a irishTitle:Rí ;
+    irishRel:nomName "Labraid";
+    owl:sameAs <#Labraid-4d52ed70>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Ní_celt_ceis>;
+    rdfs:comment "Is iarum ro gab rígi n-Érenn amal ad-fét Find m. Rossa isin fursunnud do-rigni. Gabais dano rígi fer n-Alban & do-breth Muiriath ingen Scoriath ríg Fer Morcc dó ut poeta dixit".
+
+<#Muiriath>
+    a foaf:Person;
+    irishRel:accName "Muriath";
+    irishRel:nomName "Muriath";
+    foaf:gender "female";
+    owl:sameAs <http://example.com/LL/lagin.ttl#Moriath>;
+    rel:childOf <#Scoriath>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Ní_celt_ceis>;
+    rel:spouseOf <#Labraid-b9133180>.
+
+<#Scoriath>
+    a irishTitle:Rí ;
+    irishRel:genName "Scoriath";
+    irishRel:nomName "Scoriath";
+    rdfs:comment "ríg Fer Morcc".
+
+<http://www.vanhamel.nl/codecs/Nuadu_Necht_n%C3%AD_d%C3%A1mair_anflaith>
+    prov:wasAttributedTo <#FinnFili>.
+
+<#FinnFili>
+    a foaf:Person;
+    irishRel:nomName "Finn Fili";
+    owl:sameAs <http://example.com/Rawl_B502/bunad_laigen.ttl#Find>.
+
+<http://www.vanhamel.nl/codecs/Lug_sc%C3%A9ith>
+    prov:wasAttributedTo [
+      a foaf:Person;
+      foaf:title "poeta"
+    ].
+
+<#Áine>
+    a foaf:Person;
+    irishRel:genName "Áine";
+    rel:parentOf [
+      a foaf:Person;
+      owl:sameAs <#LabraidLoingsechMóen>
+    ];
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Lug_sc%C3%A9ith>;
+    owl:sameAs <#AilellaÁine>.
+
+<#LuirccLóegaire>
+    a foaf:Person;
+    irishRel:genName "Luircc Lóegaire";
+    irishRel:nomName "Lorcc Lóegaire";
+    rel:ancestorOf [
+      a foaf:Person;
+      owl:sameAs <#LabraidLoingsechMóen>
+    ];
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Lug_sc%C3%A9ith>;
+    owl:sameAs <#LóegaireLorcc>.
+
+<#BressalBrecc>
+    a foaf:Person;
+    irishRel:nomName "Bressal Brecc";
+    irishRel:numChild 2;
+    rel:childOf <http://example.com/Rawl_B502/fursunnud_naile.ttl#BresalBrecc>;
+    owl:sameAs <http://example.com/LL/rig_ossairge.ttl#BresailBric>.
+
+<#Condla>
+    a foaf:Person;
+    irishRel:nomName "Condla";
+    rel:childOf <#BressalBrecc>;
+    rdfs:comment "senathair Ossairge ut de illis post dicemus";
+    owl:sameAs <http://example.com/LL/rig_ossairge.ttl#Condlai>.
+
+<#Lugaid>
+    a foaf:Person;
+    irishRel:nomName "Lugaid";
+    rel:childOf <#BressalBrecc>;
+    rdfs:comment "senathair Lagen";
+    owl:sameAs <http://example.com/LL/lagin.ttl#Lugaid>.
+
+<#SétnaSithbaicc>
+    a foaf:Person;
+    irishRel:nomName "Sétna Sithbaicc";
+    rel:childOf <#Lugaid>;
+    owl:sameAs <http://example.com/LL/lagin.ttl#SetnaSithbacc>;
+    rdfs:comment "ut Senchán dixit";
+    irishRel:numChild 4.
+
+<#Senchán>
+    a foaf:Person;
+    irishRel:nomName "Senchán";
+    rdfs:comment "[author of 'Cethri mc la Sétna Sithbacc']".
```

Comment:
It is in the text ("ut Senchán dixit"). Again, there is no URL for 'Cethri mc la Sétna Sithbacc' so it can't be linked to directly.